src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/DesktopOsHelper.cs

```diff
@@ -152,6 +152,52 @@ public static bool IsWin10OrServerEquivalent()
             return s_win10OrServerEquivalentLazy.Value;
         }
 
+        private static string GetRuntimeVersionInternal()
```

Comment:
Would it not be better for this logic to be in PlatformProxy? The desktop helper is not meant for mobile / uwp.

Reply:
I actually initially did go that way but because RuntimeInfo is not available in some targets, still had to do # ifs But yea, if DesktopOsHelper is not the right place semantically, I can move it.